cassandraprotocol/message/query_options.go

```diff
@@ -85,23 +91,34 @@ func WithNowInSeconds(nowInSeconds int32) QueryOptionsCustomizer {
 	}
 }
 
+func WithContinuousPagingOptions(continuousPagingOptions *ContinuousPagingOptions) QueryOptionsCustomizer {
+	return func(options *QueryOptions) {
+		options.ContinuousPagingOptions = continuousPagingOptions
+	}
+}
+
 const (
 	pageSizeNone         int32 = -1
 	DefaultTimestampNone int64 = math.MinInt64
 	NowInSecondsNone     int32 = math.MinInt32
```

Comment:
Any reason why this is `MinInt32` but `pageSizeNone` is `-1`?

Reply:
It's because a page size cannot be negative, but timestamps can, so -1 would not be a good default value for "empty" timestamps.

We need to come up with a better way to represent "absent" scalar values...

Reply:
Ah I understand now. Yeah at some point we have to implement some kind of golang implementation of `Optional<int>` or `Nullable<int>`:

```
type OptionalInt struct {
    Int            int
    HasValue bool
}
```

Reply:
very good idea, I think we can go with that!